### ISSUE TITLE
SNR estimation troubleshooting

### ISSUE THREAD

**Issue by Joselisa** [dipy/dipy#1364]
Dear Dipy experts,

I’ve just started using Dipy and I'm most interested on verifying the quality of my 30dir single-shell DTI data, so I used the SNR script (http://nipy.org/dipy/examples_built/snr_in_cc.html) and, even though it runs the following output, without showing any error, I’m afraid there is something wrong based on the corpus (attached) callosum pictures.

```
INCEs-Mac-Pro:DTI INCE$ python snr_in_cc.py 
/Users/INCE/.dipy/Teste_dipy/P00641.nii.gz
/Users/INCE/.dipy/Teste_dipy/P00641.bval
/Users/INCE/.dipy/Teste_dipy/P00641.bval
Computing brain mask...
Computing tensors...
Computing worst-case/best-case SNR using the corpus callosum...
Noise standard deviation sigma=  3.07301885497
SNR for the b=0 image is : 45.2261957834
SNR for direction 6   [ 0.99940062 -0.01646399  0.0304542 ] is : 9.03408214271
SNR for direction 7   [ 0.16625515  0.98474067 -0.05143018] is : 31.5123661156
SNR for direction 30   [ 0.07397113  0.22998713  0.97037846] is : 31.8346798525
INCEs-Mac-Pro:DTI INCE$ 
```

Any ideas? 

Thank you for your help,
Joselisa 
![cc_segmentation](https://user-images.githubusercontent.com/12244661/31901043-23cb1a9c-b7ff-11e7-9440-56861b329d52.png)
[mask_CC_part.nii.gz](https://github.com/nipy/dipy/files/1407977/mask_CC_part.nii.gz)
[mask_noise.nii.gz](https://github.com/nipy/dipy/files/1407978/mask_noise.nii.gz)

**Comment by arokem** [comment]
Looks like it's missing the CC...

You might want to tweak the "threshold" values a bit.

**Comment by Joselisa** [comment]
Dear Ariel,
Thank you very much for your suggestion. I'm not sure I did correctly, but I changed the script as follows: `threshold = (0.1, 1, 0, 0.1, 0, 0.1)` 
Now, as I visualized before and after the threshold tweak I think that the first one was better (clearly less noisy):
Before (0.6 thresholded):
![before](https://user-images.githubusercontent.com/12244661/31953299-97da7924-b8c1-11e7-9ed0-87d6d5ea6386.png)
After (0.1 thresholded):
![after](https://user-images.githubusercontent.com/12244661/31953301-995d7c88-b8c1-11e7-97b2-68f7be6a43e4.png)

Now, I'm just a bit worried about not getting the same image output as shown by DIPY documentation (below):
![image](https://user-images.githubusercontent.com/12244661/31953253-79a028e6-b8c1-11e7-883c-6d4af46c4f2d.png).

Given I can see no CC in my .png output, should I trust the SNR calculated by the script? Just in case, I'm attaching the original data. 

Thank you once again for your patience.
Cheers,
Joselisa
